atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -142,20 +144,22 @@ def run_in_worker(self):
         }
 
         operator_csv = operator_manifest.csv
-        if operator_csv:
-            self.log.info("Updating operator CSV file")
-            if not operator_csv.has_related_images():
-                self.log.info("Replacing pullspecs in %s", operator_csv.path)
-                # Replace pullspecs everywhere, not just in locations in which they
-                # are expected to be found - OCP 4.4 workaround
-                operator_csv.replace_pullspecs_everywhere(replacement_pullspecs)
-
-                self.log.info("Creating relatedImages section in %s", operator_csv.path)
-                operator_csv.set_related_images()
-
-                operator_csv.dump()
-            else:
-                self.log.warning("%s has a relatedImages section, skipping", operator_csv.path)
+        if not operator_csv:
+            raise RuntimeError("Missing ClusterServiceVersion in operator manifests")
```

Comment:
Similarly, suggest to move this check to `OperatorManifest.from_directory` as well. Thus, the code is able to express the purpose in a much straightforward way just in one place. That is, OSBS works only if there is a single CSV file existing in a container repo, it will fail when no CSV file or multiple CSV files exist.